fytProject/components/FormTrain.js

```diff
@@ -1,9 +1,16 @@
 import * as React from 'react';
+import {ONBOARDING} from '../config/settings';
+import { FileSystem } from 'expo';
 import { Animated, Dimensions, StyleSheet } from 'react-native';
 import { Button, Card, Text, View, Form, Item, Input } from 'native-base';
+const onboardingvar=ONBOARDING;

```

Comment:
Is there a reason you are assigning this here as opposed to just using the onboarding variable?